client/ayon_core/plugins/publish/collect_audio.py

```diff
@@ -46,6 +46,19 @@ class CollectAudio(pyblish.api.ContextPlugin):
     audio_product_name = "audioMain"
 
     def process(self, context):
+        # Make sure Editorial related products are excluded
```

Comment:
I don't think this belongs here. This is generic plugin for all cases, but now editorial is affecting all of them. If editorial needs different behavior for audio, it should have own collect plugin to do so.

Reply:
Now you must have at least one instance with `"audio"` in families.
1. That is change of the logic.
2. Doesn't make sense, because the instances might not be related to any other instances.

I'm still not sure I understand what is the goal of this logic? Was this added to skip audio collection if there is no audio instance?